messages/en.json

```
@@ -776,6 +776,16 @@
   "key-upload": "Upload",
   "key-write": "Write",
+  "cannot-copy-secure-key": "Cannot copy secure API key. Secure keys are only visible once during creation.",
+  "copy-and-close": "Copy and Close",
+  "create-secure-key": "Create as secure (hashed) key",
+  "create-secure-key-description": "The key will only be visible once after creation. We store only a hash - we cannot recover it later.",
+  "enforce-hashed-api-keys": "Enforce Secure API Keys",
+  "enforce-hashed-api-keys-description": "When enabled, only secure (hashed) API keys can access this organization. Plain-text API keys will be rejected.",
+  "secure-key-created": "Secure API Key Created",
+  "secure-key-hidden": "Secure key (hidden)",
+  "secure-key-warning": "This key will only be shown once. Copy it now and store it securely. You will not be able to see it again.",
+  "security-settings": "Security Settings",
```

Biome (2.1.2)

[error] The key security-settings was already declared.

This where a duplicated key was declared again.

This where a duplicated key was declared again.

If a key is defined multiple times, only the last definition takes effect. Previous definitions are ignored.

(lint/suspicious/noDuplicateObjectKeys)